src/app/mocks/categories.ts

```diff
@@ -1,11 +1,46 @@
-import { CategoryReturn } from '@/types/category';
+import { CategoryReturn, GetCategoryReturn } from '@/types/category';
```

Comment:
```suggestion
import { GetCategoryReturn } from '@/types/category';
```
If make sense, remove CategoryReturn